charts/galoy/charts/price/templates/history-cronjob.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "priceHistory.fullname" . }}-cronjob
+  labels:
+    app.kubernetes.io/managed-by: Helm
+spec:
+  schedule: "0 * * * *"
```

Comment:
some times exchanges data update take some secs/min so I think is better to run it in min 1 or 2

Reply:
Fixed